vite.config.ts

```diff
@@ -0,0 +1,14 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  plugins: [sveltekit()],
+  // https://stackoverflow.com/questions/78997907/the-legacy-js-api-is-deprecated-and-will-be-removed-in-dart-sass-2-0-0
+  css: {
+    preprocessorOptions: {
+      scss: {
+        api: 'modern-compiler',
+      },
+    },
```

Comment:
I dont think we need scss, maybe?

Reply:
uhhh. yeah ig ill just remove it. i initially thought nesting required it but i suppose i dont need it then

Reply:
you resolved this but didnt remove it